scripts/deploy.sh

```diff
@@ -71,15 +81,21 @@ elif [[ ! -z "$dtag" && -z "$ptag" && -z "$rtag" ]]; then
         exit 1
     fi
 
-    # tag it locally
+    # Merge branch into master and push to origin
+    git checkout master
+    git pull origin
+    git merge --no-ff release_$rtag -m "Merge branch devel_$rtag"
+    git push origin master
+    git branch -d release_$rtag
+
```

Comment:
Actually I do think this is necessary --- because we do need to commit/push the updated files with the updated versions. But I think `release_$rtag` needs to be `devel_$dtag` ?